### ISSUE TITLE
Remove prefixes

### ISSUE THREAD

**Issue by ipimpat** [dctrwatson/znc-fish#11]
How to "remove" the message prefixes when:

```
/msg *fish SetConfig prefix_encrypted (+)
/msg *fish SetConfig prefix_decrypted (-)
```

Doesn't allow to set "empty" values?

**Comment by DanielOaks** [comment]
Oh, I see. This is caused by my silly comparison at https://github.com/dctrwatson/znc-fish/blob/master/fish.cpp#L582 and assuming that nobody would want to set an empty value.

In that if statement, I'll probably just check the value (`prefix_en/decrypted`) and if it's a valid config option, set the appropriate 'empty' value (empty string for prefix_encrypted/decrypted, whatever else for other values)

Sorry for the trouble! I'll try to get around to it sometime soon.

**Comment by ipimpat** [comment]
Currently I work around it by editing the .registry file for each user and restarting ZNC

**Comment by DanielOaks** [comment]
Yep, that makes sense and is probably the only way to do it for now. I haven't been doing as much ZNC coding recently but I'll try to get 'round to it in the next few days and submit a PR. Thanks for submitting the issue.
